.github/workflows/changelog.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: Changelog
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - labeled
+      - unlabeled
+      - synchronize
+    branches:
+      - main
+      - stable-*
+    tags:
```

Comment:
As per my understanding, with tags set to * ,  this action will be triggered when a new tag is created.

Reply:
I don't think it works that way under the `pull_request` event. I only see in the docs that `tag` can be used with the `push` event which makes sense because you don't do a PR for a tag, you just push it. Regardless, I'm not sure we want to run all (or maybe any?) of these workflows when a tag is added...changelog in particular doesn't seem applicable to tagging.